samples/some-assertions.golo

```diff
@@ -0,0 +1,54 @@
+module some.assertions
+
+import gololang.Assertions
+
+struct Point = {
+  x,y
+}
+
+function main = |args| {
+
+  # be careful, if no callback and assertion is false, then the program is aborted
+  assert(-> 5: equals(5))
+
+  assert(-> "BoB" oftype String.class, |error| {
+    println("Huston?")
+  })
+
+  assert(predicate= -> "BoB" oftype Integer.class,
+    successMessage= "This is an Integer",
+    onSuccess= |res| {
+      println("You Win")
+    },
+    errorMessage= "This isn't an Integer",
+    onError = |error| {
+      println("Huston?")
+    }
+  )
+
+  assert(predicate= -> 5 < 0, onSuccess=|res|{}, onError=|err|{
```

Comment:
Same here and in the rest of the file, use space wisely like around operators, before a `{`, etc :wink: